packages/react-testing/src/matchers/utilities.ts

```diff
@@ -57,6 +57,49 @@ export function assertIsNode(
   }
 }
 
+export function diffs(
+  element: Element<any>[],
+  props: object,
+  expand?: boolean,
+) {
+  return element.reduce<string>(
+    (diffs, element, index) =>
+      `${diffs}${index === 0 ? '' : '\n\n'}${normalizedDiff(element, props, {
```

Comment:
I guess this was just moved, but with the way prettier makes these kind of functiony ternary-y templates awful to look at I'd almost rather just do something like

```
element.map((e) => normalizedDiff(e, props, {/*stuff*/})).join('\n\n')
```

Reply:
I'm not sure I understand, the index is relevant here, I can't just throw it away, and it doesn't make sense to pollute the `normalizedDiff` function with this (since its purpose is specific and different from this)

Reply:
I cleaned it up a bit with an intermediate variable, I think you'll like it better.